### PR TITLE
added scraper for us

### PR DESCRIPTION
**Pull request by sglyon**

### CODE REVIEW

**Comment by sglyon** [comment]
Screenshot_20210422_111132

**Comment by sglyon** [comment]
whoops, swapped variables. fixing now

**Comment by sglyon** [comment]
That's better...

Screenshot_20210422_111419

File: can_tools/scrapers/official/federal/CDC/cdc_usa_vaccines.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+                location_column="location",
+                timezone="US/Eastern",
+            )
+            .drop("variable", axis="columns")
+        )
+
+
+def one_time_backfill():
+    df = pd.read_csv(
+        "/home/sglyon/Downloads/trends_in_number_of_covid19_vaccinations_in_the_us.csv",
```
Comment (mikelehen) [inline can_tools/scrapers/official/federal/CDC/cdc_usa_vaccines.py:65]: 🤔

**Review by mikelehen — COMMENTED** [review]
Is there a reason we can't pull this from the JSON API, just like we get the state data?

https://covid.cdc.gov/covid-data-tracker/COVIDData/getAjaxData?id=vaccination_data

File: can_tools/scrapers/official/federal/CDC/cdc_usa_vaccines.py (hunk 1 above)
Comment (sglyon) [reply can_tools/scrapers/official/federal/CDC/cdc_usa_vaccines.py:65]: haha yeah I know those paths are definitely non-portable, but it is a "one time" script ;)

**Comment by sglyon** [comment]
I didn't see taht they are in the json api! thank you for pointing it out. I'll update it now

**Comment by sglyon** [comment]
Ok updated to use json api -- thanks for pointing me there @mikelehen

File: can_tools/scrapers/official/federal/CDC/cdc_vaccines.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        fetcher_url = (
+            "https://covid.cdc.gov/covid-data-tracker/COVIDData/"
+            "getAjaxData?id=vaccination_data"
+        )
+        response = requests.get(fetcher_url)
+
+        return response.json()
+
+    def _filter_rows(self, df):
+        state_abbr_list = [x.abbr for x in ALL_STATES_PLUS_DC]
```
Comment (mikelehen) [inline can_tools/scrapers/official/federal/CDC/cdc_vaccines.py:42]: It looks like CDC has Puerto Rico and Mariana Islands data.  Any idea if there's a reason we filter them out?  It's tempting to pull them in, though a separate PR would be fine if we want to reduce moving pieces.

**Review by mikelehen — APPROVED** [review]
Nice!

File: can_tools/scrapers/official/federal/CDC/cdc_vaccines.py (hunk 2 above)
Comment (sglyon) [reply can_tools/scrapers/official/federal/CDC/cdc_vaccines.py:42]: will do in new pr

**Pull request merged**